consent-ui/src/app/ais/ais-routing.module.ts

```diff
@@ -53,8 +53,8 @@ const routes: Routes = [
         component: ScaSelectPageComponent
       },
       {
-        path: ReportScaResultComponent.ROUTE,
-        component: ReportScaResultComponent
+        path: EnterScaComponent.ROUTE,
```

Comment:
this is actually EnterTanComponent to follow specs

Reply:
renamed